### ISSUE TITLE
Transform fails due to `IndexError` in middle of parsing

### ISSUE THREAD

**Issue by caufieldjh** [Knowledge-Graph-Hub/kg-phenio#70]
Transform fails in the middle of parsing `phenio.owl`:
```
13:39:57  [KGX][owl_source.py][          load_graph] WARNING: Do not know how to handle BNode: Nd63473f9110542d9b5368b973161f582
13:44:34  [KGX][rdf_source.py][              triple] WARNING: Skipping https://w3id.org/biolink/vocab/Publication http://www.w3.org/2000/01/rdf-schema#subClassOf http://org.semanticweb.owlapi/error#Error1
13:44:34  Traceback (most recent call last):
13:44:34    File "run.py", line 85, in <module>
13:44:34      cli()
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
13:44:34      return self.main(*args, **kwargs)
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/click/core.py", line 1055, in main
13:44:34      rv = self.invoke(ctx)
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
13:44:34      return _process_result(sub_ctx.command.invoke(sub_ctx))
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
13:44:34      return ctx.invoke(self.callback, **ctx.params)
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/click/core.py", line 760, in invoke
13:44:34      return __callback(*args, **kwargs)
13:44:34    File "run.py", line 62, in transform
13:44:34      kg_transform(*args, **kwargs)
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/kg_phenio/transform.py", line 37, in transform
13:44:34      t.run()
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/kg_phenio/transform_utils/phenio/phenio_transform.py", line 49, in run
13:44:34      self.parse(k, data_file, k)
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/kg_phenio/transform_utils/phenio/phenio_transform.py", line 72, in parse
13:44:34      transform(inputs=[data_file],
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/kgx/cli/cli_utils.py", line 560, in transform
13:44:34      transform_source(
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/kgx/cli/cli_utils.py", line 869, in transform_source
13:44:34      transformer.transform(input_args, output_args)
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/kgx/transformer.py", line 229, in transform
13:44:34      self.process(source_generator, sink)
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/kgx/transformer.py", line 332, in process
13:44:34      for rec in source:
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/kgx/source/owl_source.py", line 94, in parse
13:44:34      yield from self.load_graph(rdfgraph)
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/kgx/source/owl_source.py", line 170, in load_graph
13:44:34      yield from self.triple(s, p, o)
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/kgx/source/rdf_source.py", line 228, in triple
13:44:34      (element_uri, canonical_uri, predicate, property_name) = self.process_predicate(
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/kgx/source/rdf_source.py", line 591, in process_predicate
13:44:34      element = self.get_biolink_element(p)
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/kgx/source/rdf_source.py", line 847, in get_biolink_element
13:44:34      mapping = toolkit.get_element_by_mapping(predicate)
13:44:34    File "/var/lib/jenkins/workspace/ledge-graph-hub_kg-phenio_master/gitrepo/venv/lib/python3.8/site-packages/bmt/toolkit.py", line 1114, in get_element_by_mapping
13:44:34      lambda s, l: s.intersection(set(l)), without_empty_lists[1:], set(without_empty_lists[0])
13:44:34  IndexError: list index out of range
```
Since this is saying ` list index out of range` it would suggest that `bmt` is unable to get a mapping for a given predicate. Could mean there is a disconnect between the BL model `bmt` is using and what `phenio` comes in with.

Anyway, there should be a JSON conversion before this anyway.

**Comment by caufieldjh** [comment]
Wonder if it's related to this Wikidata relation that keeps following me around (and doesn't parse in `robot`, but that shouldn't be a showstopper):
```
<https://www.wikidata.org/wiki/Q2306597> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> _:genid2149661103.
```

**Comment by matentzn** [comment]
> Wonder if it's related to this Wikidata relation that keeps following me around (and doesn't parse in robot, but that shouldn't be a showstopper):

No I doubt it. This error is a known optical illusion and not an error..

**Comment by caufieldjh** [comment]
Fixed by #72 - on to the next one.
